textproc/sentencepiece/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+github.setup        google sentencepiece 0.1.84 v
+revision            0
+categories          textproc
+
+license             Apache-2
+maintainers         nomaintainer
+platforms           darwin
+
+description         Unsupervised text tokenizer for Neural Network-based\
+                    text generation.
+
+long_description    SentencePiece is an unsupervised text tokenizer\
+                    and detokenizer mainly for Neural Network-based\
+                    text generation systems where the vocabulary size\
+                    is predetermined prior to the neural model\
+                    training. SentencePiece implements subword units\
+                    (e.g., byte-pair-encoding (BPE) \[Sennrich et al.\])\
+                    and unigram language model \[Kudo.\]) with the\
+                    extension of direct training from raw\
+                    sentences. SentencePiece allows us to make a\
+                    purely end-to-end system that does not depend on\
+                    language-specific pre/postprocessing.
+
+homepage            https://github.com/google/sentencepiece
+
+checksums           rmd160  91bc4a0a23d9a4daf98abbb9ec83aed2241fdc42 \
+                    sha256  dacab5fe63875b94538e06f4995ffb8cf35400708f3c333263f7b72e2e5b43de \
+                    size    11828855
+
+use_configure   yes
+
```

Comment:
according to the website it needs a "C++11 compiler", so you likely want to set ```compiler.cxx_standard   2011```. 

Additionally, it states that ```gperftools``` is an optional library, if the idea is not to use this you should make sure (likely with an argument to cmake), that it shouldn't look for this library. Otherwise it will be used when installed and otherwise not, which will result in a non-reproducible build. Alternatively, you could add a dependency on it and make sure it actually uses it.

Reply:
Wow, that's great. Thank you! Done.